tests/assistants/openai.test.ts

```diff
@@ -0,0 +1,26 @@
+import { config } from 'dotenv';
+import { Portkey } from 'portkey-ai';
+
+config({ override: true })
+const client = new Portkey({
+    apiKey: process.env["PORTKEY_API_KEY"] ?? "",
```

Comment:
Please remove the DEV base url. We can use the default PROD url for testing SDKs.